src/main/java/com/jcabi/github/RtUser.java

```diff
@@ -138,6 +141,31 @@ public UserEmails emails() {
         return new RtUserEmails(this.ghub.entry());
     }
 
+    /**
+     * Returns all notifications of a user. Wraps the call "List your
+     *  notifications". See "List your notifications" at
+     *  https://developer.github.com/v3/activity/notifications/
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#list-your-notifications">List your notifications</a>
+     * @return Returns all notifications for this user.
+     * @todo Implement this method
+     */
+    @Override
+    public List<Notification> notifications() {
+        return new LinkedList<Notification>();
+    }
+
+    /**
+     * Marks notifications as read.
+     * @param lastread Describes the last point that notifications were
+     *  checked.
+     * @see <a href="https://developer.github.com/v3/activity/notifications/#mark-as-read">Mark as read</a>
+     * @todo Implement this method
```

Comment:
todo tag should be in format: `@todo #913:30min Description what should be done, following lines should be indented by one additional space`. Description should be written so the next person that picks this up will know what to do.